lib/Model/DVCOptions.php

```diff
@@ -7,54 +7,12 @@
  * @package  DevCycle
  */
 
-/**
- * DevCycle Bucketing API
- *
- * Documents the DevCycle Bucketing API which provides and API interface to User Bucketing and for generated SDKs.
- */
 
-namespace DevCycle\Model;
+namespace DevCycle\Api;
+
+use DevCycle\Model\DevCycleOptions;
 
 /**
- * DVCOptions Class Doc Comment
- *
- * @category Class
- * @package  DevCycle
+ * @deprecated Use DevCycle\Model\DevCycleOptions instead
  */
-class DVCOptions
-{
-    protected $enableEdgeDB;
-
-    /**
-     * Constructor
-     *
-     * @param boolean                $enableEdgeDB         flag to enable EdgeDB user data storage 
-     */
-    public function __construct($enableEdgeDB = false)
-    {
-        $this->enableEdgeDB = $enableEdgeDB;
-    }
-
-    /**
-     * Gets the enableEdgeDB flag
-     *
-     * @return boolean enable EdgeDB flag
-     */
-    public function getEnableEdgeDB()
-    {
-        return $this->enableEdgeDB;
-    }
-
-    /**
-     * Sets the enableEdgeDB flag
-     *
-     * @param boolean $enableEdgeDB 
-     *
-     * @return boolean
-     */
-    public function setEnableEdgeDB($enableEdgeDB)
-    {
-        $this->enableEdgeDB = $enableEdgeDB;
-    }
-
-}
+class_alias('DevCycle\Model\DevCycleOptions', 'DevCycle\Model\DVCOptions');
```

Comment:
@JamieSinn had to move these `class_alias` into their own files to get them to work.